ecommerce/build.gradle

```diff
@@ -22,24 +22,23 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    productFlavors {
```

Comment:
Do we need this ?

Reply:
Good catch, it seemed required to clear an error but as I can still build without it it was apparently unrelated.